sandbox/auth/session/token.go

```diff
@@ -2,16 +2,40 @@ package session
 
 import (
 	"github.com/go-jose/go-jose/v3"
+	"github.com/go-jose/go-jose/v3/jwt"
 	"golang.org/x/oauth2"
 )
 
+// Tokens can generally be ID or access. It's a bit weird to have a struct
```

Comment:
So, this struct is actually meant to represent both the id token and the access token (the access token was part of the oauth2.Token field which you removed).

It's the equivalent of what you used to call `TokenSet` in the previous library; I also considered naming it `Session`.

I ended up naming it `Token` just to match the nomenclature of `oauth2.Token`, but if you think that name is confusing, I'd be happy to go with `TokenSet` or `Session` instead.

Reply:
You are passing the ID token into both fields. Did you mean to do that? 

TokenSet or Session both make sense to me, but `Token` feels wrong if there are actually multiple tokens (id, access, refresh)

It is weird that `oauth2.Token` has access token and refresh so we need to use that in a way that makes sense for us.

A typical setup is to save a bundle that contains access_token, id_token, refresh_token, and maybe an expiration for the refresh (because refresh tokens tend to not be JWTs). The reason all of these are bundled together is because auth providers will return all of them together as part of the auth flow.